### ISSUE TITLE
AttributeError: 'AxolotlReceivelayer'

### ISSUE THREAD

**Issue by qvs5010** [AragurDEV/yowsup#55]
Hi 

does anyone know what is happening here? :(

```
  File "/home/quintin/what2/yowsup-master/yowsup/layers/axolotl/layer_receive.py", line 425, in sendKeys
    self._sendIq(setKeysIq, onResult, self.onSentKeysError)
AttributeError: 'AxolotlReceivelayer' object has no attribute 'onSentKeysError'
```

**Comment by hoehermann** [comment]
This should already be fixed by #53 in 81ac3ec.

**Comment by AragurDEV** [comment]
Fixed thanks to @hoehermann
